src/LiveDevelopment/Documents/HTMLDocument.js

```diff
@@ -263,7 +263,8 @@ define(function HTMLDocumentModule(require, exports, module) {
                 console.warn("Browser DOM does not match after change: " + JSON.stringify(change));
 
                 edits.forEach(function (delta) {
```

Comment:
Let's just get rid of this entire loop, since we're doing no work, there's no point spending cycles on it.